pywinauto/controls/UIAWrapper.py

```diff
@@ -250,6 +249,127 @@ def invoke(self):
+        iface = uia_defs.get_elem_interface(elem, "ExpandCollapse")
+        iface.Collapse()
+
+        # Return itself to allow action chaining
+        return self
+
+    #-----------------------------------------------------------
+    def get_expand_state(self):
+        """
+        An interface to CurrentExpandCollapseState property of 
+        the ExpandCollapse control pattern.
+
+        Indicates the state, expanded or collapsed, of the control.
+        
+        Values for enumeration as defined in uia_defines module:
+        expand_state_collapsed = 0
+        expand_state_expanded = 1
+        expand_state_partially = 2
+        expand_state_leaf_node = 3
+        """
+
+        elem = self.element_info.element
+        iface = uia_defs.get_elem_interface(elem, "ExpandCollapse")
+        return iface.CurrentExpandCollapseState
+
+    #-----------------------------------------------------------
+    def is_expanded(self):
+        """Test if the control is expanded"""
+        state = self.get_expand_state()
+        return state == uia_defs.expand_state_expanded
+
+    #-----------------------------------------------------------
+    def is_collapsed(self):
+        """Test if the control is collapsed"""
+        state = self.get_expand_state()
+        return state == uia_defs.expand_state_collapsed
+
+    def get_selection(self):
+        """
+        An interface to GetSelection of the SelectionProvider pattern
+
+        Retrieves a UI Automation provider for each child element
+        that is selected. Builds a list of UIAElementInfo elements 
+        from all retrieved providers.
+        """
+        elem = self.element_info.element
+        iface = uia_defs.get_elem_interface(elem, "Selection")
+        ptrs_array = iface.GetCurrentSelection()
+        return elements_from_uia_array(ptrs_array)
+
+    def can_select_multiple(self):
+        """
+        An interface to CanSelectMultiple of the SelectionProvider pattern
+
+        Indicates whether the UI Automation provider allows more than one 
+        child element to be selected concurrently.
+
+        """
+        elem = self.element_info.element
+        iface = uia_defs.get_elem_interface(elem, "Selection")
+        return iface.CurrentCanSelectMultiple
+
+    def is_selection_required(self):
+        """
+        An interface to IsSelectionRequired property of the 
+        SelectionProvider pattern.
+
+        This property can be dynamic. For example, the initial state of 
+        a control might not have any items selected by default, 
+        meaning that IsSelectionRequired is FALSE. However, 
+        after an item is selected the control must always have 
+        at least one item selected.
+        """
+        elem = self.element_info.element
+        iface = uia_defs.get_elem_interface(elem, "Selection")
+        return iface.CurrentIsSelectionRequired
+
+    def select_by_name_or_by_idx(self, item_name = None, item_index = 0):
```

Comment:
Maybe more simple name `select` is better here. It's also aligned with the pattern method name.